File: pset6/mario.py
import cs50

def main():
    while(True):
        print("n is ",end="")
        n=cs50.get_int()
        if n>=0 and n<24:
            break
        else:
            print("Please input a non-negative integer no more than 23")
    rows=n
    hashes=1
    while(True):
        if rows==0:
            break
        for i in range(rows-1):
            print(" ", end="")
        for i in range(hashes):
            print("#", end="")
        print("  ", end="")
        for i in range(hashes):
            print("#", end="")
        print()
        hashes+=1
        rows-=1



if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int row;
int i;
int main(void)
{
    do{
    printf("Height: ");
    i=get_int();
    }
    while (i>23 || i<0);
    
    int z=i-1;
    int k=2;
   for (int j=0;j<i;j++)
       {
   
    for(int l=0;l<z;l++)
    {
        printf(" ");
    }
    for(int p=0;p<k;p++)
    {
        printf("#");
    }
    
    printf("\n");
       
    z--;
    k++;
       }
    
    
    
}